tests/ga/test_regorus.py

```diff
@@ -58,42 +72,294 @@ def tearDownClass(cls):
         cls.patcher.stop()
         AgentTestCase.tearDownClass()
 
-    def test_should_evaluate_query_with_valid_params(self):
+    def test_download_allowed_should_be_true_if_extension_in_allowlist(self):
+        """
+        If extension is in the allowlist, downloadAllowed=True. Global allowlist rule shouldn't matter.
```

Comment:
NIT: maybe you can add comment on what rule you target to test

Reply:
Done